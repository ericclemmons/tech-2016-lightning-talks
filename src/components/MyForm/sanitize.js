export default function sanitize(text) {
  return text.trimLeft().trimRight();
}
