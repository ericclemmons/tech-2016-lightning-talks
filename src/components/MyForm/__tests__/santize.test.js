import sanitize from "../sanitize";

describe("santize", () => {
  it("should strip whitespace", () => {
    expect(sanitize("  test  ")).toMatchSnapshot();
  })
})
