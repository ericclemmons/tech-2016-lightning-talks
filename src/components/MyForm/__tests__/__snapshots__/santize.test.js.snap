exports[`santize should strip whitespace 1`] = `"test"`;
